README.md

```diff
@@ -79,6 +79,15 @@ The install script was tested under macOS Monterey (12.4) and Ubuntu 22.04.
 See [dependencies](DEPENDENCIES.md) for a detailed list of dependencies to use with `brew install` or `apt-get install`, depending on your platform. As compilers, we generally use recent versions of clang and gcc (Linux only). Please make sure that the system compiler points to the most recent version or use cmake (see below) accordingly.
 Older versions may work, but are neither tested nor supported.
 
+## Nix Setup
+You can build Hyrise using Nix. To do so, first [install Nix](https://nixos.wiki/wiki/Nix_Installation_Guide) on your current operating system. Afterward, run the following build in the root of the repository:
+
+```bash
+nix-shell --pure
+```
+
+This will drop you into a shell with all dependencies installed. You can now build Hyrise as usual. Please note that using the `--pure` flag is highly important as it avoids using dependencies from the local system.
```

Comment:
"highly important" > "recommended to avoid using dependencies from the local systems." ??

I don't see an issue per se with using local dependencies. It defeats the purpose of nix, right? But the current wording is not really telling why it is "highly important" and what would happen if we don't pass this flag.

Reply:
You are right about this. It is more recommended than highly important.